graphql-test-utils/cypress/add-commands.js

```diff
@@ -0,0 +1,13 @@
+import 'cypress-graphql-mock';
+import * as mocksForMc from '../graphql-mocks/mc';
+
+Cypress.Commands.add('useMockedGraphqlApi', (customOperations = {}) => {
+  cy.task('getGraphQLSchema', 'mc').then(schema => {
+    cy.mockGraphql({
+      schema,
+    });
+  });
+  const operations = mocksForMc.createMockOperations(customOperations);
+  cy.mockGraphqlOps({ operations });
+  return cy.wrap(operations);
```

Comment:
Do you know what this does?

Reply:
Yes, it allows to get the value like this:

```js
cy.useMockedGraphqlApi().then(operations => {
})
```

Reply:
Otherwise it won't work, because Cypress queues everything that happens in the commands and executes them asynchronously.

Reply:
Ah this auto queueing promisification. Got it.